mitosheet/src/components/taskpanes/ExcelRangeImport/ExcelRangeSheetSelection.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import DropdownItem from "../../elements/DropdownItem";
+import Input from "../../elements/Input";
+import Select from "../../elements/Select";
+import Col from "../../layout/Col";
+import Row from "../../layout/Row";
+import { ExcelRangeImportParams } from "./ExcelRangeImportTaskpane";
+
+interface ExcelRangeSheetSelectionProps {
+    sheet_name: string | undefined;
+    sheet_names: string[],
+    params: ExcelRangeImportParams,
+    setParams: React.Dispatch<React.SetStateAction<ExcelRangeImportParams>>
+    
+}
+function ExcelRangeSheetSelection(props: ExcelRangeSheetSelectionProps): JSX.Element {
+
+    const params = props.params;
+    const setParams = props.setParams;
+    const sheet_name = props.sheet_name;
+
+    return (
+        <>
+            <Row justify="space-between" align="center">
+                <Col>
+                    <p className="text-header-3">
+                            Find Sheet By
```

Comment:
Maybe instead of "Find" we use "Select". IMO symmetry here is helpful in grouping these together.